### PR TITLE
Update "Realm" to "App Services" in cards & toc

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
- Name change.
- Added a separate "Realm SDKs" sidebar entry. No card, since then we'd end up with a row with only one card in it and it would look a bit wonky.

### CODE REVIEW

**Review by sumedhamehta — APPROVED** [review]
nice!

**Review by sumedhamehta — APPROVED** [review]

File: source/index.txt [hunk 1]
```diff
@@ -85,7 +85,7 @@ Welcome to the MongoDB Documentation
    .. card::
       :headline: Build a mobile app
       :cta: Get started with MongoDB Realm
```
Comment (schmalliso) [inline source/index.txt:87]: do you want to update the cta as well here?

**Review by schmalliso — COMMENTED** [review]
LGTM % one suggested change that I've framed as a question for some reason :D

File: source/index.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/index.txt:87]:
```suggestion
      :cta: Get started with Atlas App Services
```

**Pull request merged**